utils/setup.sh

```diff
@@ -58,24 +58,31 @@ function remove_tmp_dir {
 }
 trap remove_tmp_dir EXIT
 
+function error_exit {
+    exit 1
+}
+
```

Comment:
Do we want to just `set -e` at the top? Or are we selectively exiting?

Reply:
There's a couple of places where I test if roles or policies already exist, and handle the case where they don't. I'm generally a fan of `set -e`, but it definitely complicates the case where failure is OK. In this case, I figured that explicit error handling was less problematic... Although it was only a hair's breadth ahead of "rewrite it in Python" :wink:

Reply:
Can probably just adjust those cases by adding something like `|| true` to them. I guess we just have to pick our poison :)

Reply:
I see, yea you're right, we'd need really long lines practically speaking. I think it's fine the way it is, was mostly just curious if it was an alternative.